locales/ru/validation.php

```diff
@@ -14,39 +14,39 @@
 return [
     'accepted'             => 'Вы должны принять :attribute.',
     'accepted_if'          => 'Поле :attribute должно быть принято, когда :other соответствует :value.',
-    'active_url'           => 'Поле :attribute содержит недействительный URL.',
-    'after'                => 'В поле :attribute должна быть дата больше :date.',
-    'after_or_equal'       => 'В поле :attribute должна быть дата больше или равняться :date.',
-    'alpha'                => 'Поле :attribute может содержать только буквы.',
-    'alpha_dash'           => 'Поле :attribute может содержать только буквы, цифры, дефис и нижнее подчеркивание.',
-    'alpha_num'            => 'Поле :attribute может содержать только буквы и цифры.',
-    'array'                => 'Поле :attribute должно быть массивом.',
-    'before'               => 'В поле :attribute должна быть дата раньше :date.',
-    'before_or_equal'      => 'В поле :attribute должна быть дата раньше или равняться :date.',
+    'active_url'           => 'Значение поля :attribute не является действительным URL.',
+    'after'                => 'Значение поля :attribute должно быть датой после :date.',
+    'after_or_equal'       => 'Значение поля :attribute должно быть датой после или равной :date.',
+    'alpha'                => 'Значение поля :attribute может содержать только буквы.',
+    'alpha_dash'           => 'Значение поля :attribute может содержать только буквы, цифры, дефис и нижнее подчеркивание.',
+    'alpha_num'            => 'Значение поля :attribute может содержать только буквы и цифры.',
+    'array'                => 'Значение поля :attribute должно быть массивом.',
+    'before'               => 'Значение поля :attribute должно быть датой до :date.',
+    'before_or_equal'      => 'Значение поля :attribute должно быть датой до или равной :date.',
     'between'              => [
         'array'   => 'Количество элементов в поле :attribute должно быть между :min и :max.',
         'file'    => 'Размер файла в поле :attribute должен быть между :min и :max Килобайт(а).',
         'numeric' => 'Значение поля :attribute должно быть между :min и :max.',
         'string'  => 'Количество символов в поле :attribute должно быть между :min и :max.',
     ],
     'boolean'              => 'Поле :attribute должно иметь значение логического типа.',
-    'confirmed'            => 'Поле :attribute не совпадает с подтверждением.',
-    'current_password'     => 'Поле :attribute содержит некорректный пароль.',
-    'date'                 => 'Поле :attribute не является датой.',
-    'date_equals'          => 'Поле :attribute должно быть датой равной :date.',
-    'date_format'          => 'Поле :attribute не соответствует формату :format.',
+    'confirmed'            => 'Поле :attribute не совпадает с подтверждаемым.',
+    'current_password'     => 'Некорректный пароль.',
```

Comment:
```suggestion
    'current_password'     => 'Значение поля :attribute содержит некорректный пароль.',
```

В данном файле все сообщения об ошибках обязательно должны ссылаться на конкретное поле.

Reply:
Ориентировался на то, что есть в english, там название атрибута не подставляется: _The password is incorrect._

Reply:
Тейлор много странностей вносит, что ж, оставим.